src/building/how-to-build-and-run.md

```diff
@@ -78,12 +78,12 @@ You can install it with `cargo install --path src/tools/x`.
 
 ## Building the Compiler
 
-To build a compiler, run `./x.py build`. This will build up to the stage1 compiler,
+To build the compiler, run `./x.py build`. This will build up to the stage1 compiler,
```

Comment:
I think "a" was more clear. There are multiple stages; `build` only builds one of them.